pkgs/applications/audio/friture/default.nix

```diff
@@ -1,16 +1,16 @@
-{ lib, fetchFromGitHub, python3Packages, wrapQtAppsHook }:
+{ lib, fetchFromGitHub, python39Packages, wrapQtAppsHook }:
 
 let
-  py = python3Packages;
+  py = python39Packages;
```

Comment:
Please revert this. python3 points to python39.

Reply:
Neat, I wasn't aware of this